rocprim/include/rocprim/intrinsics/atomic.hpp

```diff
@@ -45,6 +45,12 @@ namespace detail
         return ::atomicAdd(address, value);
     }
 
+    ROCPRIM_DEVICE ROCPRIM_INLINE
+    float atomic_add(double * address, double value)
```

Comment:
Shouldn't the result be double?

Reply:
Indeed! Looks like I copy-pasted too fast. I've fixed this up, thanks.